pkgs/tools/security/openpgp-card-tools/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, rustPlatform
+, fetchCrate
+, pkg-config
+, pcsclite
+, nettle
+, darwin
+, testers
+, openpgp-card-tools
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "openpgp-card-tools";
+  version = "0.0.12";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-3OKOMe7Uj+8qpzfu0DzqwIGa/QJ0YoKczPN9W8HXJZU=";
+  };
+
+  cargoHash = "sha256-gq17BXorXrlJx4zlvLuOT8XGUCqZXFDSxgs/Fv9dChk=";
+
+  nativeBuildInputs = [ pkg-config rustPlatform.bindgenHook ];
+  buildInputs = [ pcsclite nettle ]
+    ++ lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.PCSC ];
```

Please pass in PCSC via an argument to `callPackage`, see how other packages do it in all-packages for an example.

```suggestion
    ++ lib.optionals stdenv.isDarwin [ PCSC ];
```